apps/modernization-ui/src/pages/addPatient/components/LeftBar/LeftBar.tsx

```diff
@@ -22,14 +22,14 @@ export const LeftBar = ({ activeTab }: any) => {
                     New patient
                 </h6>
             </div>
-            <div className="border-base-light grid-row flex-no-wrap border-top padding-y-1 margin-x-3 cursor-pointer width-full">
+            {/* <div className="border-base-light grid-row flex-no-wrap border-top padding-y-1 margin-x-3 cursor-pointer width-full">
```

Comment:
Is this commented out code left in for a reason?  If so can you add that reason to the comment.
The same for all the commented out code.